cockatrice/src/window_main.cpp

```diff
@@ -1031,6 +1031,11 @@ void MainWindow::actAddCustomSet()
         return;
     }
 
+    if (QFileInfo(fileName).suffix() != "xml") { // fileName = *.xml
```

Comment:
Could we prevent non-xml files from being shown to the user at all by using `dialog.setNameFilters` ?